adobe_vipm/flows/fulfillment/transfer.py

```diff
@@ -1284,6 +1288,48 @@ def __call__(self, client, context, next_step):
         next_step(client, context)
 
 
+class ManageAccountRevival(Step):
+    """Manages the account revival."""
+
+    def __call__(self, client, context, next_step):
+        """Manages the account revival."""
+        adobe_client = get_adobe_client()
+        customer_subscriptions = adobe_client.get_subscriptions(
```

Comment:
Mmm.. since we are dealing only with the main agreement here, it is better to filter subscriptions that belongs only to the main agreement